### PR TITLE
feat: added support to check aurora AcuUtilization instead rds free s…

### PR DESCRIPTION
**Pull request by mccar**
Ticket - https://oat-sa.atlassian.net/browse/REL-917

## Goal
Support has been added to monitoring the Aurora AcuUtilization system when the program works with it.

## Changelog
- feat: added support to check aurora AcuUtilization instead rds free space

## How to test
1. Open Tools -> System status page to see the system status monitor
2. There is 'Used space on RDS storage' for RDS instances and 'ACU Utilization on RDS storage' for Aurora cluster instances
3. You can check unit tests for Aurora system status checker `AwsRDSAcuUtilizationCheckTest`

### CODE REVIEW

File: model/Check/System/AwsRDSFreeSpaceCheck.php [hunk 1]
```diff
@@ -69,23 +56,27 @@ protected function doCheck(): Report
...
     }
 
     /**
      * @return bool
+     * @throws SystemCheckException
      */
     public function isActive(): bool
     {
-        return $this->isAws();
+        $instanceData = $this->getInstanceData($this->getStackId($this->getRDSHost()));
```
Comment (vbyndych) [inline model/Check/System/AwsRDSFreeSpaceCheck.php]: **question**: How we can get RDS instance data in a non-AWS environment? I'm asking as we first pull instance data and only then check if it is AWS or not.

File: model/Check/System/AwsRDSAcuUtilizationCheck.php [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    private const ID_ACU_UTILIZATION = "acuutilization";
+    private const NAMESPACE = 'AWS/RDS';
+    private const METRIC_NAME = 'ACUUtilization';
+
+    /**
+     * @inheritdoc
+     */
+    protected function doCheck(): Report
+    {
+        $rdsHost = $this->getRDSHost();
```
Comment (vbyndych) [inline model/Check/System/AwsRDSAcuUtilizationCheck.php]: **suggestion**: as we do not use the output of those functions anywhere here except getInstanceData, can we move those function calls there?

**Review by vbyndych — APPROVED** [review]

**Comment by vbyndych** [comment]
**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code respects code style rules
- [x] New code respects best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

File: model/Check/System/AwsRDSAcuUtilizationCheck.php (hunk 2 above)
Comment (mccar) [reply model/Check/System/AwsRDSAcuUtilizationCheck.php]: fixed

File: model/Check/System/AwsRDSFreeSpaceCheck.php (hunk 1 above)
Comment (mccar) [reply model/Check/System/AwsRDSFreeSpaceCheck.php]: fixed

**Review by augustas — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**